src/_constants/texts.js

```diff
@@ -22,6 +22,9 @@ export default ln => {
     const relatedText = t[ln];
     for (var key in relatedText) {
         if (relatedText.hasOwnProperty(key) && relatedText[key] instanceof Array) {
+            if(relatedText[key][1] == " ")
```

Comment:
@aminroosta I am not sure if this is what you wanted to do `relatedText[key][1] = relatedText[key][1].trim()`. There could be multiple white spaces?

Reply:
@arnabk yea, closed the pr in favor of your changes ... this pr was not a solution.